src/rdsn/src/meta/meta_backup_engine.cpp

```diff
@@ -246,7 +246,31 @@ void meta_backup_engine::on_backup_reply(const error_code err,
         }
     }
 
-    // TODO(heyuchen): handle other status
+    if (response.status == backup_status::UPLOADING) {
+        ddebug_f(
+            "backup_id({}): backup for partition {} from server {} is uploading, progress = {}",
+            _cur_backup.backup_id,
+            pid.to_string(),
+            primary.to_string(),
+            response.upload_progress);
+    }
+
+    if (response.status == backup_status::SUCCEED) {
+        ddebug_f("backup_id({}): backup for partition {} from server {} succeed",
```

Comment:
log seem can be combine one code for `uploading` and `succeed`:
```c++
dedebug(.... status=xxx...)

if (success) {
.....
}
```

Reply:
I think it is better to seperate them, you can reference the whole function.